doc/examples/segmentation/plot_hausdorff_distance.py

```diff
@@ -3,80 +3,70 @@
 Hausdorff Distance
 ==================
 
-This example shows how to calculate the Hausdorff distance between two sets of
-points. The `Hausdorff distance
+This example shows how to calculate the Hausdorff distance between a
+"ground truth" and a "predicted" segmentation mask. The `Hausdorff distance
 <https://en.wikipedia.org/wiki/Hausdorff_distance>`__ is the maximum distance
-between any point on the first set and its nearest point on the second set,
-and vice-versa.
+between any point on the contour of the first mask and its nearest point on the
+contour of the second mask, and vice-versa.
 
+It can be computed either directly from the segmentation masks using
+``hausdorff_distance_mask`` or from contours images (i.e. binary images where
```

Comment:
```suggestion
``hausdorff_distance_mask`` or from their respective contours (i.e., binary images where
```
I'm anticipating the conventions we'll come up with for the [typed API](https://github.com/scikit-image/skimage-archive/blob/main/grants/2022_CZI_EOSS5/full_application.md#description-of-typed-api-work): Technically, a contour (any binary image) would be a "boolean mask" array (maybe we'll agree on naming it just a "mask" array).